hw_3/task_1.js

```diff
@@ -0,0 +1,13 @@
+//=====================================
+for(i =10; i>=0; i--) {
```

Comment:
Explicit variable declaration
```suggestion
for(let i =10; i>=0; i--) {
```